charts/base/templates/_keda.tpl

```diff
@@ -26,8 +26,12 @@ spec:
     {{- if .Values.keda.scaleTargetRef.envSourceContainerName }}
     envSourceContainerName: {{ .Values.keda.scaleTargetRef.envSourceContainerName }}
     {{- end }}
-  pollingInterval: {{ .Values.keda.pollingInterval | default 30 }}
-  cooldownPeriod: {{ .Values.keda.cooldownPeriod | default 300 }}
+  {{- if .Values.keda.pollingInterval }}
```

Comment:
the problem with those that keda will set default values regardless, therefore it might be misleading to make it optional

Reply:
Yes, I want the keda defaults & not predefined defaults. It adds more complexity as it has a default value here & a default value in `values.yaml` & neither are needed.